qiita_pet/support_files/doc/source/processingdata/qp-fastp-minimap2.rst

```diff
@@ -0,0 +1,37 @@
+Adapter and host filtering
+==========================
+
+At the end of August 2023, we discovered that the parameters used by
+qp-fastp-minimap2 did not trigger application of adapter filtering. By default,
+fastp performs autodetection of adapters and filtering for single-end data. By
+default, fastp does not perform these operations on paired-end data. This behavior
+was not expected by us. It was discovered when manually assessing replicated
+sequences, which on examination by BLAST against NT reported to be adapters.
+
+Adapter filtering for paired-end data with fastp requires specifying either the
+exact adapters to remove (i.e., no autodetection), or to explicitly specify “--detect_adapter_for_pe”. Qiita previously indicated to users that the
```

Comment:
This line appears to run longer than the others. I think it's meant to break after 'specify'?

Reply:
Agreed but that's merely a non written or enforced rule ... I think is fine but if you feel strongly about it I can add a new line.

Reply:
Nope! no worries on my end.